qemu-check-zephyr.exp

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/expect -f
+#
+# This scripts starts QEMU, loads and boots Linux/OP-TEE, then runs
+# tests in the guest. The return code is 0 for success, >0 for error.
+#
+# Options:
+#   -q        Suppress output to stdout (quiet)
+#   --timeout Timeout for each test (sub)case, in seconds [480]
```

Comment:
I may be wrong here because I do not know how to program `.exp`, but if `[480]` is a default value then it differs from line 15 `set timeout 900`.